generated/nidaqmx/_grpc_interpreter.py

```diff
@@ -2260,6 +2260,36 @@ def load_task(self, session_name):
             metadata=metadata)
         return response.task, response.new_session_initialized
 
+    def perform_bridge_shunt_cal_ex(
+            self, task, channel, shunt_resistor_value,
+            shunt_resistor_location, shunt_resistor_select,
+            shunt_resistor_source, bridge_resistance,
+            skip_unsupported_channels):
+        response = self._invoke(
+            self._client.PerformBridgeShuntCalEx,
+            grpc_types.PerformBridgeShuntCalExRequest(
```

Comment:
These RPC and requests do not exist in nidaqmx.proto.

Reply:
This will cause a runtime error. Do you think that is good enough, or should this explicitly throw some sort of `NotSupported` error?

Reply:
We shouldn't ship code that uses nonexistent methods and types. This would fail with static analysis tools like mypy.

We should do one of the following:
- (preferred) Add these methods to grpc-device and nidaqmx.proto. This will return a gRPC `UNIMPLEMENTED` status with older versions of the NI gRPC Device Server and work with newer versions of the server.
- Update the codegen metadata to indicate that these methods are not supported for gRPC.